src/mission_companion.cpp

```diff
@@ -429,6 +429,11 @@ bool talk_function::display_and_choose_opts( mission_data &mission_key, const tr
 
     reset_cur_key_list();
 
+    if( cur_key_list.empty() ) {
+        popup( _( "There are no missions at this colony.  Press Spacebar..." ) );
```

Comment:
Maybe reword to "Press any key..."?

Reply:
This is the same message as at the beginning of the function.
https://github.com/CleverRaven/Cataclysm-DDA/blob/bcb6e0ec90e9d1e10f14bcd7b6e68c854b485c34/src/mission_companion.cpp#L368
IMO, better to leave it as it is to save translations.